README.md

```diff
@@ -55,19 +63,19 @@ updates those lists.
 The commit cache has to be created manually:
 ```
 $ ./pasta sync # Creates cache file for commits on the patch stacks
-$ ./pasta sync -mbox # Update / synchronise mailboxes before creating caches
+$ ./pasta sync -mbox # Update / synchronise mailboxes before creating caches if in mail box mode
 ```
 
 ### Detecting and grouping similar patches
-Detecting similar patches on patch stacks (i.e., branches) and eventually
+Detecting similar patches on patch stacks (i.e. branches) or mail boxes and eventually
 linking them into equivalence classes is split in two different commands:
 `pasta analyse` and `pasta rate`.
 Reason for the split is the comparatively long duration of the analysation
 phase. After `pasta analyse`, you might want to reuse the results of the
 analysation and run `pasta rate` for several times on the same data set.
 
-The detection phase is split in four steps:
-1. Comparing successive versions on the patch stacks
+The detection phase is split in three steps:
+1. If in Patch Stack mode, comparing successive versions on the patch stacks
```

Comment:
In patch stack mode, the comparison of successive ...

Reply:
Sure